src/main/java/duke/utility/Parser.java

```diff
@@ -0,0 +1,48 @@
+package duke.utility;
+
+import duke.command.*;
+import duke.exception.DukeException;
+
+import java.util.StringTokenizer;
+
+/**
+ * Parser for user input
+ */
+public class Parser {
+    Parser(){
+    }
+
+    public static Command parse(String inputCommand) throws DukeException {
+        StringTokenizer commandTokenizer = new StringTokenizer(inputCommand);
+        Command command = null;
+
+        switch(commandTokenizer.nextToken()){
+            case "list":
```

Comment:
Should these case clauses not be indented?
<img width="1056" alt="Screenshot 2022-02-04 at 11 16 49 PM" src="https://user-images.githubusercontent.com/52398668/152553974-75d7fc93-504b-4682-bfc9-d59d0dfcf29e.png">
.